src/plugins/integrations/AWSIntegration.ts

```diff
@@ -200,7 +204,6 @@ class AWSIntegration implements Integration {
                 [SpanTags.OPERATION_TYPE]: operationType ? operationType : 'READ',
```

Comment:
we might use empty operation type instead of `READ` if the operation is not known

Reply:
Done